services/meta/association.md

```diff
@@ -3,29 +3,40 @@ This service allows creating a direct association between a source device (e.g.
 
 This service intends to replace `group` commands in `dev_sys` service.
 
-A controllable device (like a lamp) would have a list of `in_services` that would indicate the services it supports. A controller (like a button) would have a list of `out_services` instead.
```

Comment:
Possibly put here a block 

```
> In Zigbee a controllable...
```

Reply:
Moved to the services props since it's connected to them